runelite-client/src/main/java/net/runelite/client/plugins/interfacestyles/InterfaceStylesPlugin.java

```diff
@@ -129,6 +136,48 @@ public void onConfigChanged(ConfigChanged config)
 		}
 	}
 
+	@Subscribe(priority = 9)
```

Comment:
Usually we comment why the priorities are set, what is this trying to run after?